resize_data_part.target-sh

```diff
@@ -27,15 +28,23 @@ echo "Unmounting the partition..."
 /bin/mount -o remount,ro /media
 # Try to remount the data partition on error (early EXIT is an error).
 trap mount_data_partition EXIT
+echo
 
 echo "Updating partition table..."
-
-# If the --no-reread option is not supported, this command will fail (|| true takes care of that)
-echo "${START},${SIZE}" | sfdisk --no-reread -uS -N ${PART_NUM} /dev/${DEVICE} || true
-
-clear # Previous command has very noisy output
-echo "Resizing the data partition. Please be patient."
+SFDISK_OPTS="--no-reread --quiet -uS -N ${PART_NUM}"
+# Older versions of sfdisk do not support --no-tell-kernel.
```

Comment:
I don't think you need to care about older versions of sfdisk. If OpenDingux' sfdisk supports --no-tell-kernel, then use it unconditionally.

Reply:
Agreed

Reply:
Done